### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by bokobza**

### CODE REVIEW

**Comment by bokobza** [comment]
image

**Comment by fassadlr** [comment]
@bokobza awesome! #greenbuilds4eva

File: appveyor.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ test_script:
             Push-Location $testFolder
 
             Write-Host "[$env:configuration] Running tests in $testFolder" -foregroundcolor "magenta"
-            dotnet test -c $env:configuration
+            dotnet test -c $env:configuration --no-build
```
Comment (fassadlr) [inline appveyor.yml]: What does this do?

Comment (bokobza) [reply appveyor.yml]: @fassadlr it prevents rebuilding the dlls you just built :-), altogether gaining a few seconds ;-)

**Pull request merged**